app/panels.php

```diff
@@ -208,6 +208,84 @@
       'disabled' => 0,
     ),
     array (
+      'key' => "{$ns}_default_location",
+      'label' => 'Map Default Location',
```

Comment:
What's a better name here.

"Default" = It isn't really default as there is a default fallback, it's more like "Centred"/"Starter"/"Initial"

"Location" to me just sounds like there's one whereas it's a repeater field, so it should be plural

so maybe `Map Initial Locations`? Pick one of them and update all other references throughout your PR